kong/kong.lua

```diff
@@ -239,6 +240,13 @@ function Kong.balancer()
       tostring(addr.ip),"' port:",tostring(addr.port),"): ", tostring(err))
     return responses.send_HTTP_INTERNAL_SERVER_ERROR()
   end
+
+  ok, err = set_timeouts(addr.connect_timeout / 1000,
+                         addr.send_timeout / 1000,
+                         addr.read_timeout / 1000)
```

Comment:
> Can't we do these calculations at config-load time, instead of on each request?

I'd rather not. It'd be harder to track in case something goes wrong or if we start loading APIs from another place and forget to do the conversion, etc... 

> Should be an easy fix with the new caching (the way I intend to do it)

I think we should sync up on that, because the way I understood it, I was supposed to work on the new caching in 0.11? BTW, I cannot know which way you intend to do it if you do not elaborate.